telegramer/core.py

```diff
@@ -688,6 +695,20 @@ def apply_label(self, tid):
     def update_stats(self):
         log.debug('update_stats')
 
+    def checkSpeed(self):
+        log.debug("Minimum speed: %s" , self.config['minimum_speed'])
+        if self.config['minimum_speed'] <= -1:
+                return
+        try:
+            for t in component.get('TorrentManager').torrents.values():
+                if t.status.state == 3 :
```

Comment:
Consider using:
 self.list_torrents(lambda t: t.get_status(('state',))['state'] == 'Downloading')
as used in cmd_down, instead of magic number 3.

Reply:
Tried that before, it seems I can't get it to work:
```
for x in self.list_torrents(lambda t : t.get_status(('state',))['state'] == 'Downloading')
```
Gives me a Syntax error, am I missing something obvious?